credentials/alts/internal/handshaker/handshaker.go

```diff
@@ -154,39 +156,33 @@ type altsHandshaker struct {
 	side core.Side
 }
 
-// NewClientHandshaker creates a ALTS handshaker for GCP which contains an RPC
+// NewClientHandshaker creates an ALTS handshaker for GCP which contains an RPC
 // stub created using the passed conn and used to talk to the ALTS Handshaker
 // service in the metadata server.
```

Comment:
`contains an RPC stub created using the passed conn` seems to be internal detail (which isn't true either, because what is stored internally is the grpc.ClientConn and the associated stream). Could we avoid including that in the doctstring. Here and in `NewServerHandshaker`.

Reply:
Rewrote this paragraph for clarity.